### PR TITLE
[ADD] added new color for pgn and user name, added link in main menu

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: chess/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+For Odoo 8
+
+Usage
+=====
+
+**Interface**
+
+* 1 Chess board
+* 2 Съеденные фигуры
+* 3 История ходов в возможностью просмотра ходов
```
Comment (yelizariev) [inline chess/doc/index.rst]: move to README

File: chess/doc/index.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* 4 Автоничья, если недостаточно фигур для мата.
+* 5 Один из игроков сдался.
+* 6 Один из игроков принял ничью.
+* 7 У одного из игроков кончилось время.
+
+**Результат игры**
+
+Результатом игры является новый рейтинг игрока, который вычисляется по системе Elo, и записывается слева от шахматной доски.
+
+**Note.** To use the module, you need to be sure that your odoo instance support longpolling, i.e. Instant Messaging works.
```
Comment (yelizariev) [inline chess/doc/index.rst]: add link to this page https://odoo-development.readthedocs.io/en/latest/admin/longpolling.html

File: chess/doc/index.rst (hunk 1 above)
Comment (GabbasovDinar) [inline chess/doc/index.rst]: @yelizariev ты имеешь ввиду раздел 
**Interface**
перенести в README?

Comment (yelizariev) [inline chess/doc/index.rst]: да

**Pull request merged**